docs/source/best-practices/workquota.md

```diff
@@ -0,0 +1,30 @@
+(work-directory-storage-quota)=
+# Work Directory Storage Quota
+
+Every space in work has a quota for TB and inodes (file count). It is necessary to stay below the quota to be able to use the space effectively. 
+
+We recommend the following practices to ensure effective use of your space in `/work`
+
+1. [Check your usage](https://rc-docs.northeastern.edu/en/latest/best-practices/homequota.html#how-to-check-your-quotas) regularily to avoid hitting the quota limit in the middle of running jobs, which will disrupt their execution. 
+
+2. Regularly compress directories
```

Comment:
Regularly compress directories that are not used frequently or for projects that have been completed.